docs/source/release-process.rst

```diff
@@ -126,23 +126,20 @@ Edit your ``~/.pypirc`` file to look like:
    [distutils]
    index-servers =
        pypi
-       pypitest
 
    [pypi]
    username:
    password:
 
-   [pypitest]
```

Comment:
This seems to not work anymore. The only way I could get pypitest to work was using the incantation below, which came from https://packaging.python.org/guides/using-testpypi/.